.azure-pipelines/generation-templates/objc.yml

```diff
@@ -5,3 +5,9 @@ steps:
     BuildConfiguration: $(buildConfiguration)
     OutputFullPath: $(typewriterDirectory)/output
     RepoModelsDir: $(Build.SourcesDirectory)/msgraph-sdk-objc-models/GeneratedModels/
+- pwsh: |
+      gem install --user-install xcodeproj
+      & chmod +x ./add_generated_models.rb
```

Comment:
Do we need this line? I think the executable in next step is `ruby` so I don't think we need to convert the script itself to be an executable.